packages/main/src/TextArea.ts

```diff
@@ -80,11 +80,10 @@ type ExceededText = {
  * <code>import "@ui5/webcomponents/dist/TextArea";</code>
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.TextArea
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-textarea
+ * @extends UI5Element
+ * @implements {IFormElement}
```

Comment:
With implements tag we only describe public interfaces like IBar, IAvatar and etc.. so this one is redundant.